app/core/env.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace app\core;
+use Exception;
+
+/**
+ * @throws Exception
+ */
+function loadEnv($path = __DIR__ . '/../.env'): void
```

Comment:
Is this function required?  Can the variables from the `.env` file just be read from by using `$_ENV`?

Reply:
Yeah, I think I should be able to read the env through $_ENV directly. I know the best practice in Laravel is kinda using a dependecy called dotenv to read the env file directly. Since I did not have composed installed, I just followed similar pattern as class autoloader and using View class to render all the views.

Reply:
Yeah, dotenv is a pretty handy package :laughing: